mkdocs.yaml

```diff
@@ -57,10 +57,11 @@ nav:
     - Licenses: community/licenses.md
     - History: community/history.md
     - Partnerships:
-      - NWB: community/partnerships/nwb.md
-      - DANDI: community/partnerships/dandi.md
+      - Open Ephys GUI: community/partnerships/openephysgui.md
       - Suite2p: community/partnerships/suite2p.md
       - Facemap: community/partnerships/facemap.md
+      - NWB: community/partnerships/nwb.md
+      - DANDI: community/partnerships/dandi.md
       - INCF: community/partnerships/incf.md
```

Comment:
Do we have a reasoning behind the order here? Should we alphabetize?

Reply:
Good idea.  Updated.